app_test.go

```diff
@@ -89,6 +90,30 @@ func TestNewApp(t *testing.T) {
 		assert.Contains(t, errMsg, "depends on fx_test.A")
 		assert.Contains(t, errMsg, "depends on fx_test.B")
 	})
+
+	t.Run("Visualizer", func(t *testing.T) {
+		found := false
+		fixture := "fxtest/graph.dot"
+		type A struct{}
+		type B struct{}
+		var g DotGraph
+		app := fxtest.New(t,
+			Option(Visualizer()),
+			Provide(func() A { return A{} }),
+			Provide(func() B { return B{} }),
```

Comment:
Make a dependency that changes the graph to make it clear what you are attempting to test.

Reply:
Specifically, the packages that you are Providing to the fx app should be actual dependencies. The examples you provide don't exercise the logic of dig visualize.

Reply:
Done.

Reply:
Done.